pydata_sphinx_theme/static/css/theme.css

```diff
@@ -1,3 +1,5 @@
+@import "../basic.css";
```

Comment:
where does `basic.css` come from? I can't find it in the repository so I am confused as to how this works

Reply:
Inherited from the parent theme (basic.css is already included in the assets, just higher down the line).